### DISCUSSION TITLE
Nudge users to sign petitions for non-existing electricity data APIs

### DISCUSSION THREAD

**Discussion by fbarl** [electricitymaps/electricitymaps-contrib#2262] [Ideas]
I had a discussion with @corradio about what the community could do to incentivize more electricity providers to open up / build the public APIs for their electricity data and he mentioned it might be a good idea to direct users to sign a petition demanding it.

The way I imagined this to work is by embedding a link for creating / signing a petition in the zone details panel in the UI for each zone currently verified to have no public data sources (something like a verified _stage 1_ zone in https://github.com/tmrowco/electricitymap-contrib/issues/1977#issuecomment-588964185). We'd probably have to have the information about which organizations are supposed to be in charge for the petition to make sense.

On the implementation side, my first thought was to use [Change.org](https://www.change.org/), but it seems they're just in the process of [taking down their API](https://help.change.org/s/article/Change-org-API?language=en_US), so perhaps an alternative platform like [actionnetwork](https://actionnetwork.org/docs/v2/) would work. Having a petition API would make things much easier for us, but something could also be worked out without it I guess.

**Comment by corradio** [comment]
cc @martincollignon / @sorensvejstrup as we could re-use this concept in North!
